networkx/algorithms/components/connected.py

```diff
@@ -160,6 +166,12 @@ def node_connected_component(G, n):
     NetworkXNotImplemented
         If G is directed.
 
+    Examples
+    --------
+    >>> G = nx.Graph([(0, 1), (1, 2), (5, 6), (3, 4)])
+    >>> nx.node_connected_component(G, 0)  # nodes of component having '0' node
```

Comment:
Just a minor wording suggestions to avoid possible confusion of node `0` being written as a string in the comment.

```suggestion
    >>> nx.node_connected_component(G, 0)  # nodes of component that contains node 0
```